submarine-vis/src/components/App.js

```diff
@@ -16,11 +17,39 @@ class App extends Component {
       isSafetyConditionAchieved: true,
       submarineRegion: 'yellow',
       trenchAlert: 'red',
-      probes: [96, 4, 7],
+      probes: [],
+      turns: [],
     };
   }
 
-  // TODO: have a function that talks to the server for game stats
+  componentDidMount() {
+    this.interval = setInterval(() => {
```

Comment:
I.... _suppose_ this is okay. Though I'd prefer if `interval` was a part of our state.

Reply:
I just changed it to be part of our state. I'm not proficient in javascript so please take a look and let me know how I messed it up.